docs/docsite/rst/playbook_guide/playbooks_handlers.rst

```diff
@@ -5,6 +5,10 @@ Handlers: running operations on change
 
 Sometimes you want a task to run only when a change is made on a machine. For example, you may want to restart a service if a task updates the configuration of that service, but not if the configuration is unchanged. Ansible uses handlers to address this use case. Handlers are tasks that only run when notified.
 
+.. note::
+
+    Ansible sets "changed" status by itself, but you can also define the rules using ``changed_when``. See :ref:`override_the_changed_result` to determine it explicitly.
```

Comment:
Rather than a note it might be worth adding a section or extending an existing section, like "Controlling when handlers run". As it is now, this doesn't really explain how `changed_when` relates to handlers.

This could be useful in a situation where you want to control when a handler runs by manually setting the "changed" status. I guess there could be instances when you'd want to notify a handler if the main task doesn't make any changes too.

The `playbooks_handlers.rst` file has some examples of notifying handlers to restart services so you could maybe keep it consistent with those and add something along the lines of:

```
- name: Copy httpd configuration
   ansible.builtin.copy:
     src: ./new_httpd.conf
     dest: /etc/httpd/conf/httpd.conf
   # The task is always reported as changed
   changed_when: True
   notify: Restart apache
```

This could be useful to ensure the handler restarts the service every time the new configuration gets copied. Maybe if the src and dest files are identical it wouldn't be marked as a change. You could probably confirm that with a quick test.